Letterbook.Adapter.Db/DependencyInjection.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Npgsql;
+
+namespace Letterbook.Adapter.Db;
+
+public static class DependencyInjection
+{
+	public static IServiceCollection AddDbAdapter(this IServiceCollection services, IConfigurationSection config)
+	{
+
+		var dbOptions = config.Get<DbOptions>() ?? throw new ArgumentNullException(DbOptions.ConfigKey);
+
+		return services.AddDbContext<RelationalContext>(options => options.UseNpgsql(DataSource(dbOptions)));
+	}
```

Comment:
And we accomplish that by configuring the `DbContextOptionsBuilder` as part of calling `AddDbContext` in DI.